Source: me#gmail.com
Upstream: No, upstream won't accept this.
Reason: No strip, install icons to /usr/share/pixmaps.
--- gcin-1.4.5.pre4.orig/Makefile	2009-03-29 21:32:52.000000000 +0800
+++ gcin-1.4.5.pre4/Makefile	2009-04-18 19:37:57.000000000 +0800
@@ -218,8 +218,8 @@
 	install $(GCIN_SO) $(gcinlibdir)
 
 install:
-	install -d $(datadir)/icons
-	install -m 644 gcin.png $(datadir)/icons
+	install -d $(datadir)/pixmaps
+	install -m644 gcin.png $(datadir)/pixmaps
 	$(MAKE) -C icons install
 	install -d $(gcinlibdir)
 	install $(GCIN_SO) $(gcinlibdir)
@@ -230,7 +230,7 @@
 	if [ $(QT_IM) = 'Y' ]; then $(MAKE) -C qt-im install; fi
 	if [ $(QT4_IM) = 'Y' ]; then $(MAKE) -C qt4-im install; fi
 	if [ $(prefix) = /usr/local ]; then \
-	   install -m 644 gcin.png /usr/share/icons; \
+	   install -m 644 gcin.png /usr/share/pixmaps; \
 	   install -d $(DOC_DIR); \
 	   install -m 644 README.html Changelog.html $(DOC_DIR); \
 	   install $(PROGS) $(bindir); \
@@ -238,7 +238,7 @@
 	else \
 	   install -d $(DOC_DIR_i); \
 	   install -m 644 README.html Changelog.html $(DOC_DIR_i); \
-	   install -s $(PROGS) $(bindir); \
+	   install $(PROGS) $(bindir); \
 	   rm -f $(bindir)/trad2sim; ln -sf sim2trad $(bindir)/trad2sim; \
 	fi
 	$(MAKE) -C scripts install
